isstools/widgets/qas/widget_general_info.py

```diff
@@ -0,0 +1,14 @@
+from PyQt5 import uic, QtGui, QtCore
+import pkg_resources
+
+import isstools.widgets.widget_general_info as widget_general_info
+
+ui_path = pkg_resources.resource_filename("isstools", "ui/ui_general_info_qas.ui")
```

Comment:
Instead of using the BL names, can we name it by the functionality involved? Such as "basic info", "info with amplifiers", and so on? In case of a third beamline is added, which only needs the basic info widgets, we won't need a separate module to support it. What do you think?

Reply:
I don't think there any chance that another beamline will adopt this code in the near future and I like being able to see at the file and directory level where BL-specific code can be found. I also think we will be able to move faster if we just divide the current code between ISS and QAS and leave further refactoring for later.

I do think we could work the existing code into some reusable widgets and I tried to do that with the general info widget at first. I created two amplifier widgets but I was not able to figure out how to easily make QtDesigner work with them. That was the reason I went with separate general info widgets and general info uis. I think the best way to work with different amplifier widgets would be to lay them out programmatically. If we want to develop more reusable widgets I think that will mean more programmatic layouts.

Reply:
I see, that all sounds reasonable, and I agree with the ISS/QAS split first, and then refactoring/generalizing the code later.